tools/pontoon/translate.md

```diff
@@ -0,0 +1,82 @@
+
+#### Manually Switch to Suggestion Mode
+
+To manually switch to Suggestion Mode, click the Settings gear to the lower-left side of the editing space and click the icon to the left of 'Make Suggestions' so that it becomes a green circle. The `Suggest` button should then appear to the lower-right side of the editing space.
+
+![Editing space settings](/assets/images/pontoon/translate/suggest_settings.png) ![Activate make suggestions](/assets/images/pontoon/translate/activate_suggest.png)
+![Make suggestions activated](/assets/images/pontoon/translate/activated_suggest.png)
+
+A notification will appear at the top of the window, indicating that `Make Suggestions` has been enabled.
+
+![Translation saved notification](/assets/images/pontoon/translate/translation_saved_notif.png)
+
+To switch to Review and Direct Translation Mode, go to the Settings menu by clicking on the gear and turn off the `Make Suggestions` option. The green `Save` button should appear to the lower-right side of the editing space.
+
+![The green save button](/assets/images/pontoon/translate/save_button.png)
+
+### Review a Suggestion
+
+To review a suggestion, search for suggested strings or click on a suggested string in the sidebar.
+
+If the suggested string is an acceptable translation, click on the `Save` button to the lower-right side of the editing space. This will approve the suggestion as a translation and save the string to the localized file.
+
+If the suggested string is **not** an acceptable translation, turn on `Make Suggestions` in settings. Once the `Suggest` button appears on the lower-right side of the editing space, edit the suggested string as appropriate and click `Suggest`. This will update the suggested string in the Pontoon database but will not save the string in the localized file. To see the old and new suggestions, click on the history tab below the editing space.
+
+![History Tab](/assets/images/pontoon/translate/history_tab.png)
+
+### Submit a Translation Directly
+
+*Note that it is always best practice to have a translation reviewed by another translator; however, if no reviewer is available, these instructions allow a translator to translate without review.*
+
+To submit a translation directly, search for and click on any string in the sidebar.
+
+Ensure that the `Make Suggestions` option in `Settings` is disabled. Then input the translation to the editing space and click `Save`. This will save your translation directly, without review, to the localized file.
+
+## Translating in the In-context Editor
+
+*Note that only websites that have been enabled for in-context editing will display in context in Pontoon*
+
+To translate in context, find the string you want to translate in the in-context view and double click it or click on the pencil icon on the top-left of the string. Hovering over the string that you want to translate will highlight that string in the sidebar and in the in-context view.
```

Comment:
Hovering in this context is transitive as far as I know
https://www.merriam-webster.com/dictionary/hover